### PR TITLE
Movies I hw

### PR DESCRIPTION
**Pull request by meganmckeough**

### CODE REVIEW

File: Megan/wk05/1-mon/notes/pry(main)> HTTParty.get('http:::omdbapi.com:?t=jaw [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ "Type"=>"movie",
+ "DVD"=>"11 Jul 2000",
+ "BoxOffice"=>"N/A",
+ "Production"=>"Universal Pictures",
+ "Website"=>"http://www.jaws25.com/",
+ "Response"=>"True"}
+[3] pry(main)> results = HTTParty.get('http://omdbapi.com/?t=jaws&apikey=2f6435d9');
+[4] pry(main)> results["Year"]
+=> "1975"
+[5] pry(main)>
```
Comment (Sonna) [inline Megan/wk05/1-mon/notes/pry(main)> HTTParty.get('http:::omdbapi.com:?t=jaw]: Strange filename `pry(main)> HTTParty.get('http:::omdbapi.com:?t=jaw`

Comment (meganmckeough) [reply Megan/wk05/1-mon/notes/pry(main)> HTTParty.get('http:::omdbapi.com:?t=jaw]: I know, it autosaved that and I didn't realise! (...and "notes" just seemed so generic haha)
I'll change it.

Comment (meganmckeough) [reply Megan/wk05/1-mon/notes/pry(main)> HTTParty.get('http:::omdbapi.com:?t=jaw]: It's not for use, just some notes I saved.

Comment (Sonna) [reply Megan/wk05/1-mon/notes/pry(main)> HTTParty.get('http:::omdbapi.com:?t=jaw]: I know, I was looking at reviewing people's most recent code for any glaring issues. This is a non-issue, a strange anomaly and was not sure if you knew given the filename.

**Pull request merged**